src/ui/MissingTerm/MissingTerms.ts

```diff
@@ -141,7 +141,7 @@ export class MissingTerms extends Component {
 
   private buildMissingTerms(): Dom[] {
     const terms: Dom[] = this.missingTerms.map(term => {
-      if (this.containsFeaturedResults(term) || this.containsWildcard(term)) {
+      if (this.containsFeaturedResults(term) || this.containsWildcard(term) || this.constainsApostrophe(term)) {
```

Comment:
I think it would be clearer to first do a `filter` operation to get only the valid terms, then afterward a `map` to create the clickable element itself, instead of doing both at the same time.

I would also rework the condition a bit to extract a `private isValidTerm(term: string)` method, which chains all those sub conditions.

So we get 
```
const validTerms = this.missingTerms.filter(term => this.isValidTerm(term));
const terms = validTerms.map(validTerm => this.makeTermClickableIfEnabled(term));
```